editor/editor_export_godot3.cpp

```diff
@@ -532,6 +532,14 @@ void EditorExportGodot3::_rename_properties(const String &p_type, List<ExportDat
 		if (prop_rename_map.has(E->get().name)) {
 			E->get().name = prop_rename_map[E->get().name];
 		}
+
+		/* Hardcoded fixups for properties that changed definition in 3.0 */
```

Comment:
Maybe there would be a better place for those fixups?
cc @reduz, I can implement the actual list of fixes, but I'm not against some advice on how to structure that code.